pages/adventure/adventure-style.css

```
@@ -1,66 +1,78 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,900;1,700&family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap');
 
-html {
-  background-color: #E1EDFF;
-}
-
 body {
   font-family: "roboto", sans-serif;
   color: #f2f2df;
+  background-color: #E1EDFF;
 }
 
-.navbar ul li {
-  font-family: "roboto", sans-serif;
+.navbar {
   text-align: center;
+}
+
+.navbar__ulist {
```

`ulist` sounds a bit confusing. What does "U" stand for? You should use more descriptive, semantic names.

```suggestion
.navbar__ulist {
```